translations/open-status.pot

```diff
@@ -0,0 +1,23 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+
+msgctxt "Whether a store is open or closed"
```

Comment:
what do you think about something like: "The hours that a store is open"

Reply:
updated!